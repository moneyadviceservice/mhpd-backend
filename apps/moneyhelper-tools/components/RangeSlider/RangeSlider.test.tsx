import { render } from '@testing-library/react';
import { RangeSlider } from '.';

describe('RangeSlider component', () => {
  it('renders correctly', () => {
    const { container } = render(<RangeSlider />);
    expect(container).toMatchSnapshot();
  });
  it('renders with label', () => {
    const { container } = render(<RangeSlider label="Select a value" />);
    expect(container).toMatchSnapshot();
  });
});
