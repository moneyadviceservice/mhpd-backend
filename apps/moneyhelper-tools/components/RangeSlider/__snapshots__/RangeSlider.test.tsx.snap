// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RangeSlider component renders correctly 1`] = `
<div>
  <div>
    <input
      class="w-full mt-4 mb-3 tool-slider"
      max="100"
      min="0"
      step="1"
      type="range"
      value="0"
    />
    <div
      aria-hidden="true"
      class="flex justify-between"
    >
      <p>
        £0
      </p>
      <p>
        £100
      </p>
    </div>
  </div>
</div>
`;

exports[`RangeSlider component renders with label 1`] = `
<div>
  <div>
    <label>
      Select a value
    </label>
    <input
      class="w-full mt-4 mb-3 tool-slider"
      max="100"
      min="0"
      step="1"
      type="range"
      value="0"
    />
    <div
      aria-hidden="true"
      class="flex justify-between"
    >
      <p>
        £0
      </p>
      <p>
        £100
      </p>
    </div>
  </div>
</div>
`;
